sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/implementation/AmqpSendLink.java

```diff
@@ -45,4 +45,11 @@
      * Gets the context for this AMQP send link.
      */
     ErrorContext getErrorContext();
+
+    /**
+     * The remote endpoint path this link is connected to.
+     *
+     * @return The remote endpoint path this link is connected to.
+     */
+    String getHostname();
```

Comment:
This hostname is common to an AmqpLink. Update azure-core-amqp's AmqpLink to have this method